juniper_warp/Cargo.toml

```diff
@@ -8,21 +8,19 @@ documentation = "https://docs.rs/juniper_warp"
 repository = "https://github.com/graphql-rust/juniper"
 edition = "2018"
 
-[features]
-async = [ "juniper/async", "futures03" ]
-
 [dependencies]
-warp = "0.1.8"
+warp = "0.2"
 juniper = { version = "0.14.2", path = "../juniper", default-features = false  }
 serde_json = "1.0.24"
 serde_derive = "1.0.75"
-failure = "0.1.2"
+failure = "0.1.7"
 # TODO: rebase juniper_warp to futures 3 once warp supports it
 futures = "0.1.29"
```

Comment:
Because warp 2.0 uses futures, all the futures-0.1 stuff should be removed. We should be able to remove the old futures and rename futures03